A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -3322,6 +3328,61 @@ class FMCMainDisplay extends BaseAirliners {
     isAltitudeManaged() {
         return SimVar.GetSimVarValue("AUTOPILOT ALTITUDE SLOT INDEX", "number") === 2;
     }
+
+    updateTransitionAltitude(icao, phase) {
+        NXApi.getAirport(icao)
+            .then((data) => {
+                this.transitionAltitude = data.transAlt;
+                if (this.transitionAltitude === -1) {
+                    if (phase === "origin") {
+                        SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", 18000);
+                        return;
+                    }
+                    if (phase === "destination") {
+                        SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", 18000);
+                        return;
+                    }
+                } else {
+                    if (phase === "origin") {
+                        SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", this.transitionAltitude);
+                        return;
+                    }
+                    if (phase === "destination") {
+                        SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", this.transitionAltitude);
+                        return;
+                    }
+                }
+            });
```

Comment:
We should probably `.catch()` here in case the airport doesn't exist or the network request fails on the server, just so no Javascript error occurs.

Reply:
Ok!

Reply:
I added it. can you check it?